### DISCUSSION TITLE
Update Several rows in a table

### DISCUSSION THREAD

**Discussion by taholmes160** [Budibase/budibase#9179] [Help]
HI Folks:

I have a table that looks like this:
![image](https://user-images.githubusercontent.com/24684691/210104775-7274ba16-eba2-4b12-9717-b0b0a7862965.png)

I want to be able to select several rows of the table and update the Current Location column with a new value

My initial way to attack this is to use an UPDATE query against the table (this is a mysql database) but I am not at all familiar with actions, so there might be a better way

I have been trying to figure out how to pass the selections via the url (but im not sure thats the best way) but I cant seem to get them to pass either

Can someone point me in the right direction -- or in the direction of a doc that will show me what to do?

Thanks
TIM

**Comment by melohagan** [comment]
Hey @taholmes160 

Have you had a look at this answer, it might help: https://github.com/Budibase/budibase/discussions/8759#discussioncomment-4207094

**Comment by taholmes160** [comment]
Thanks Mel -- that looks useful, I'll have a go at it laters
